doc/sphinx-guides/source/installation/config.rst

```diff
@@ -226,6 +229,39 @@ You also have the option to set a custom container name separator. It is initial
 
 ``./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.files.swift-folder-path-separator=-"``
 
+Amazon S3 Storage
++++++++++++++++++
+
+For institutions and organizations looking to use Amazon's S3 cloud storage for their installation, we can do can this with some brief setup steps. 
+
+You'll need an AWS account with an associated S3 bucket for your installation to use. From the S3 management console, you can poke around and get familiar with your bucket. Make note of the bucket's name and the region its data is hosted in. It's recommended you install the CLI tool `pip <https://pip.pypa.io//en/latest/>`_ to install the `AWS command line interface <https://aws.amazon.com/cli/>`_ if you don't have it.
+
+First, we'll get our access keys set up. If you already have your access keys configured, skip this step. From the command line, run:
+
+``pip install awscli``
+
+``aws configure``
+
+You'll be prompted to enter your Access Key ID and secret key, which should be issued to your AWS account. The subsequent config steps after the access keys are up to you.
```

Comment:
Amazon stores keys in `~/.aws/credentials` and the current functionality with Dataverse has no trouble finding said credentials there. For consistency with AWS docs and possible access troubleshooting information users may need, we considered it easiest to leave the access keys where Amazon puts them. 

Also, if the individual setting up the installation is in charge of S3 stuff, they may already have configured their access keys / may have pre-existing access keys in `~/.aws/credentials`.